packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -398,3 +396,89 @@ async function sendConversionResult(
   };
   await sqs.sendMessage(sendParams).promise();
 }
+
+async function storePreProcessedConversionResult({
+  conversionResult,
+  conversionResultFilename,
+  message,
+  lambdaParams,
+  log,
+}: {
+  conversionResult: FHIRBundle;
+  conversionResultFilename: string;
+  message: SQSRecord;
+  lambdaParams: Record<string, string | undefined>;
+  log: typeof console.log;
+}) {
+  try {
+    await executeWithRetries(
+      () =>
+        s3Utils.s3
+          .upload({
+            Bucket: conversionResultBucketName,
+            Key: conversionResultFilename,
+            Body: JSON.stringify(conversionResult),
+            ContentType: "application/fhir+json",
+          })
+          .promise(),
+      {
+        ...defaultS3RetriesConfig,
+        log,
+      }
+    );
+  } catch (error) {
+    console.log(`Error uploading conversion result: ${error}`);
+    capture.error(error, {
+      extra: {
+        message,
+        ...lambdaParams,
+        conversionResultFilename,
+        context: lambdaName,
+        error,
+      },
+    });
+  }
+}
+
+async function storePreConversionPayloadInS3({
+  payload,
+  preConversionFilename: preProcessedFilename,
+  message,
+  lambdaParams,
+  log,
+}: {
+  payload: string;
+  preConversionFilename: string;
+  message: SQSRecord;
+  lambdaParams: Record<string, string | undefined>;
+  log: typeof console.log;
+}) {
+  try {
+    await executeWithRetries(
+      () =>
+        s3Utils.s3
+          .upload({
+            Bucket: conversionResultBucketName,
+            Key: preProcessedFilename,
+            Body: JSON.stringify(payload),
+            ContentType: "application/xml",
```

Comment:
nit: we have constants defined for this - MIME_TYPE_XML or something like that

Reply:
done